tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -360,6 +360,33 @@ public function adminDeletesUserUsingTheGraphApi(string $user, ?string $byUser =
 		);
 	}
 
+	/**
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function deleteTheUserUsingTheGraphApi(string $user):void {
+		// Always try to delete the user
+		// users can be deleted using the username in the GraphApi too
+		$this->adminDeletesUserUsingTheGraphApi($user);
+
+		// Only log a message if the test really expected the user to have been
+		// successfully created (i.e. the delete is expected to work) and
+		// there was a problem deleting the user. Because in this case there
+		// might be an effect on later tests.
+		if ($this->featureContext->theUserShouldExist($user)
+			&& (!\in_array($this->featureContext->getResponse()->getStatusCode(), [200, 204]))
+		) {
+			\error_log(
+				"INFORMATION: could not delete user '$user' "
+				. $this->featureContext->getResponse()->getStatusCode() . " " . $this->featureContext->getResponse()->getBody()
+			);
+		}
+
+		$this->featureContext->rememberThatUserIsNotExpectedToExist($user);
```

Comment:
method `adminDeletesUserUsingTheGraphApi` should return the response.
```suggestion
	public function deleteTheUserUsingTheGraphApi(string $user): ResponseInterface {
		// Always try to delete the user
		// users can be deleted using the username in the GraphApi too
		$response = $this->adminDeletesUserUsingTheGraphApi($user);

		// Only log a message if the test really expected the user to have been
		// successfully created (i.e. the delete is expected to work) and
		// there was a problem deleting the user. Because in this case there
		// might be an effect on later tests.
		if ($this->featureContext->theUserShouldExist($user)
			&& (!\in_array($response->getStatusCode(), [200, 204]))
		) {
			\error_log(
				"INFORMATION: could not delete user '$user' "
				. $response->getStatusCode() . " " . $response->getBody()
			);
		}

		return $response;
```